src/python/WMCore/WMSpec/StdSpecs/TaskChain.py

```diff
@@ -477,7 +495,7 @@ def modifyTaskConfiguration(self, taskConf,
                                        taskConf.get("FilterEfficiency")
 
         if taskConf["EventsPerJob"] is None:
-            taskConf["EventsPerJob"] = int((8.0 * 3600.0) / (taskConf.get("TimePerEvent", self.timePerEvent)))
+            taskConf["EventsPerJob"] = int((8.0 * 3600.0) / taskConf.get("TimePerEvent", self.timePerEvent))
```

Comment:
Don't change this, but of course there is no reason to use 8.0 vs 8, etc. That was done to force the old code into being float division.